index.html

```diff
@@ -376,14 +369,15 @@ <h2>Automatic Commands</h2>
           </li>
           <li>
             `Heart reactions` - The bot will check the ID of each message's
-            author. If the ID is in the array provided in the `love` config
-            setting, the bot will add a random heart react to the message.
+            author. If the ID is in the array provided in the
+            `|config add/remove hearts [@user]` config setting,
+            the bot will add a random heart react to the message.
           </li>
         </ul>
       </section>
 
       <section id="cmd_man">
-        <h2>Manual Commands</h2>
+        <h2>Manual Commands (60 commands available)</h2>
```

Comment:
```suggestion
        <h2>Manual Commands</h2>
```

Let's remove this, to avoid having to update every time a new feature is added.